microsoft/hyper-v/default.nix

```diff
@@ -0,0 +1,104 @@
+{ config, pkgs, options, lib, ... }:
+
+with lib; {
+
+  options = {
+    hyperv.enhancedSession = mkOption
```

Comment:
I think these things are general enough to be put into nixpkgs itself.
We have `nixpkgs/nixos/modules/virtualisation/hyperv-guest.nix`.
This repo is more for hardware-specific quirks that can be expressed with minimal amount of nixos module options.
While your contribution is provides configuration for a hypervisor target than can be easier test by a wider audience.

Could you open a pull request towards nixpkgs? If you put a link here. I can have a look.

Reply:
As there was a `hyper-v` folder on this project I thought it was the appropriate place. I could share latter on nixpkgs instead (but likely on a weekend).

Reply:
I think we could still have a simple module that imports the right nixpkgs module. Just so that users can find easier find it, when browsing nixos-hardware.